OCSDataView/R/get_data_view_interpolated.R

```diff
@@ -3,8 +3,8 @@ get_data_view_interpolated <- function(data_access_endpoint, tenant_id, namespac
   library(jsonlite)
 
   # build the correct URL from the provided properties
-  dataview_req_url <- paste(data_access_endpoint, "/tenants/", tenant_id, "/namespaces/", namespace_id, "/DataViews/", dataview_id,
-                            "/Data/Interpolated?startIndex=", start_index, "&endIndex=", end_index, "&interval=", interval, sep = "")
+  dataview_req_url <- paste(URLencode(data_access_endpoint), "/tenants/", URLencode(tenant_id), "/namespaces/", URLencode(namespace_id), "/DataViews/", URLencode(dataview_id),
```

Comment:
Can't you just do all the string concatenation inside the URLencode call? Like:
`URLencode(paste(data_access_endpoint, "/tenants/", tenant_id, "/namespaces/".....`

Reply:
that makes a lot more sense. At first I was only encoding the indexes, so I wrapped the individual pieces, but then decided to do all parameters. Not sure how this didnt occur to me :|